### PR TITLE
Init implementation of Chartist control powered by ReactBridge

### PR DESCRIPTION
**Pull request by adamjez**
First draft of **Chartist Control** with use of **ReactBridge**
The control is based on [ChartistJs](https://gionkunz.github.io/chartist-js) and [React Wrapper](https://react.rocks/example/react-chartist)

Chartist control supports three types: **line, pie, bar**
Samples of usage are located in Sample2.dothml

Missing features (bugs):
+ You can't add function into control Options eg. to dynamically create labels
+ You can't create animations like in the Chartist Control [samples](https://gionkunz.github.io/chartist-js/examples.html#example-line-path-animation)

### CODE REVIEW

File: Controls/ReactBridge/src/DotVVM.Contrib/ChartData.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    {
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public IEnumerable<string> labels { get; set; }
+        [Bind(Direction.None)]
+        public IEnumerable<string> Labels { get => labels; set => labels = value; }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public IList<IEnumerable<double>> series { get; set; }
+        [Bind(Direction.None)]
+        public IList<IEnumerable<double>> Series { get => series; set => series = value; }
```
Comment (exyi) [inline Controls/ReactBridge/src/DotVVM.Contrib/ChartData.cs:17]: :) This is a fancy trick, I'd just like to say, that as of dotvvm 1.2 it's possible to client-side rename property using Bind(Name = "series")

File: Controls/ReactBridge/src/DotVVM.Contrib/Chartist.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        protected override void OnInit(IDotvvmRequestContext context)
+        {
+            context.ResourceManager.AddRequiredResource("ReactChartist");
+
+            var reactBridge = new ReactBridge
+            {
+                Name = "ReactChartist.default",
+            };
+
+            reactBridge.Attributes["type"] = Type.ToString();
```
Comment (exyi) [inline Controls/ReactBridge/src/DotVVM.Contrib/Chartist.cs:66]: You can use `reactBridge.Attributes["type"] = this.GetValueRaw(TypeProperty)` to allow binding in `Type` property

**Review by exyi — APPROVED** [review]

File: Controls/ReactBridge/src/DotVVM.Contrib/Chartist.cs (hunk 2 above)
Comment (adamjez) [reply Controls/ReactBridge/src/DotVVM.Contrib/Chartist.cs:66]: okay, I didn't know that, thanks

File: Controls/ReactBridge/src/DotVVM.Contrib/ChartData.cs (hunk 1 above)
Comment (adamjez) [reply Controls/ReactBridge/src/DotVVM.Contrib/ChartData.cs:17]: Good to know

**Pull request merged**